testing/test_bounding_box.py

```diff
@@ -22,7 +22,6 @@
             "log_directory": LOG_DIR
         },
         "transforms_params": {
-            "Resample": {"wspace": 0.75, "hspace": 0.75},
```

Comment:
Not needed anymore?

Reply:
Before the changes, Resample was not applied as preprocessing for this test, but now that it is automatically being applied as preprocessing, the bounding box dimensions were not good anymore since the dimensions corresponded to the resampled image :). This transform was not necessary for this test.